README.md

```diff
@@ -52,6 +52,27 @@ To stop database services:
 3. Elevate your privs: `User.new_super_admin("your NetID here")`
 
 ### Development with AWS resources
+During development, by default, files are stored using the Local file system. This is configured in `./config/storage.yml`:
+
+```
+local: &local
+  service: 'Disk'
+  root: <%= Rails.root.join("storage") %>
+
+development: &development
+  <<: *local
+```
```

Comment:
I'd worry about keeping multi-line extracts in sync: Maybe just point users to `storage.yaml`, and keep the details there?